src/main/java/com/theelixrlabs/healthcare/constants/DoctorPatientAssignmentConstants.java

```diff
@@ -8,6 +8,7 @@ public class DoctorPatientAssignmentConstants {
     public static final String DOCTOR_ALREADY_ASSIGNED_KEY = "doctor.already.assigned";
     public static final String DOCTOR_ALREADY_UNASSIGNED_KEY = "doctor.already.unassigned";
     public static final String DOCTOR_ID_INVALID_KEY = "doctor.id.invalid";
```

Comment:
Rename DOCTOR_ID_INVALID_KEY to INVALID_ DOCTOR_ID_KEY

Reply:
Renamed DOCTOR_ID_INVALID_KEY to INVALID_ DOCTOR_ID_KEY.